docs/templates/plugin.rst.j2

```diff
@@ -106,7 +106,7 @@ Parameters
             <th width="100%">Comments</th>
         </tr>
         {% for key, value in options|dictsort recursive %}
-            <tr>
+            <tr id="@{ key }@">
```

Comment:
I would change this slightly to:
```suggestion
            <tr id="parameter-@{ key }@">
```
That way there won't be a collision if `key` happens to equal another anchor's name.